File: 1967. Number of Strings That Appear as Substrings in Word/solution.cpp
class Solution {
public:
  int numOfStrings(vector<string>& patterns, string word) {
    int count = 0;
    for (int index = 0; index < patterns.size(); index++){
      string pattern = patterns[index];
      if (word.find(pattern) != -1){
        count += 1;
      }
    }

    return count;
  }
};

File: 1967. Number of Strings That Appear as Substrings in Word/solution.js
/**
 * @param {string[]} patterns
 * @param {string} word
 * @return {number}
 */
 var numOfStrings = function(patterns, word) {
  let count = 0;
  for (let index = 0; index < patterns.length; index++){
    let pattern = patterns[index];
    if (word.includes(pattern)){
      count += 1;
    }
  }

  return count;
};